msteams-platform/toolkit/add-capability.md

```diff
@@ -8,103 +8,88 @@ ms.topic: overview
 
-*   You can also open the command palette and enter Teams: Add Capabilities:
+  > [!Note]
+  > You need to provision for each environment, after you have successfully added the capabilities in your Teams app.
 
-    :::image type="content" source="../assets/images/teams-toolkit-v2/manual/tree view capabilities.png" alt-text="Alternate capabilities":::
+* **To add capabilities by using Teams Toolkit in Visual Studio Code:**
 
+   1. Open **Visual Studio Code**.
+   1. Select **Teams Toolkit** from left panel.
+   1. Select **Add features** under **DEVELOPMENT**.
 
-    1. From the pop-up, select the capabilities to include in your project:
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/manual/select-feature1234.png" alt-text="updated one" border="true":::
 
-    :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/select capabilities.png" alt-text="select":::
+* **To add capabilities by using command palette:**
 
-    2. Select **OK**
+   1. Open **command palette**.
+   1. Enter **Teams:Add features**.
+   1. Press **Enter**.
 
-The selected capabilities are successfully added to your project. The Teams Toolkit generate source code for newly added capabilities
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/manual/Teams-add-features.png" alt-text="team feature" border="true":::
 
-## Add capabilities using TeamsFx CLI in command window
+   1. From the pop-up, select the capability to add in your project.
 
-1. Change directory to your **project directory**
-1. Execute the following command to add different capabilities to your project:
+       :::image type="content" source="~/assets/images/teams-toolkit-v2/manual/notification-add-capabilities.png" alt-text="notification" border="true":::
 
-   |Capability and Scenario| Command|
-   |-----------------------|----------|
-   |To add tab|`teamsfx capability add tab`|
-   |To add bot|`teamsfx capability add bot`|
-   |To add message extension|`teamsfx capability add messaging-extension`|
+## Add capabilities using TeamsFx CLI
+
+* Change directory to your **project directory**
+* The following table lists the capabilities and required commands:
+
+  |Capability and Scenario| Command|
+  |-----------------------|----------|
+  |To add notification bot |`teamsfx add notification `|
+  |To add command bot |`teamsfx add command-and-response `|
+  |To add sso-enabled tab |`teamsfx add sso-tab`|
+  |To add tab |`teamsfx add tab`|
+  |To add bot |`teamsfx add bot`|
+  |To add message extension |`teamsfx add message extension`|
 
 ## Supported capabilities
 
-Apart from the capabilities your Teams app already have, you can choose to add different capabilities to your Teams app. The following table provides the different Teams app capabilities:
+You can choose to add different capabilities in your Teams app.
+The following table lists the different Teams app capabilities:
 
```

Comment:
the above two sentences are not precise. Maybe we should explicitly tell user that depends on different project they have created, their options when adding addtional capabilities are different. Below is the table list of available/meaningful capability to add to their project.

Reply:
Updated as mentioned.